app/views/coronavirus_landing_page/components/shared/_timeline.html.erb

```diff
@@ -2,22 +2,70 @@
   <%= render "govuk_publishing_components/components/heading", {
     text: timeline["heading"],
     padding: true,
-    margin_bottom: 4
+    margin_bottom: 2
   } %>
 
-  <% if timeline["summary"] %>
-    <%= render_govspeak(timeline["summary"]) %>
-  <% end %>
-
-  <% timeline["list"].each do | item | %>
-    <div class="covid-timeline__item">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: item["heading"],
-        heading_level: 3,
-        font_size: "s",
-        margin_bottom: 1,
+  <% if show_timeline_nations?(timeline["list"]) %>
+    <%= form_tag coronavirus_landing_page_path,
+        method: :get,
+        class: "js-change-location",
+        data: {
+          module: "gem-track-click",
+        } do |f| %>
+      <%= render "govuk_publishing_components/components/radio", {
+        name: "timeline_nation",
+        heading: "Choose location",
+        items: timeline_nations_items(@selected_country)
       } %>
-      <%= render_govspeak(item["paragraph"]) %>
+      <div class="covid-timeline__button-wrapper">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "View",
+          margin_bottom: 6
+        } %>
+      </div>
+    <% end %>
+
+    <div aria-live="polite">
+      <% CoronavirusTimelineNationsHelper::UK_COUNTRY_LIST.each do |nation| %>
+        <div id="nation-<%= nation %>" class="js-covid-timeline <%= 'covid-timeline__item--hidden' unless display_country(@selected_country) == nation %>">
```

Comment:
Since this has `covid-timeline__item--hidden` we should have `covid-timeline__item` set on it as well.

Reply:
@andysellick ☝️ 🥺

Reply:
If you want to follow BEM convention to the letter, then yes, I suppose so. Problem is that there are no other styles required on this element other than to hide it, so the suggested additional class wouldn't provide any styles. I can look at renaming it to something more appropriate.

Reply:
Yes please to take a look. I found myself a bit confused at the existence of a modifier without the thing it's modifying existing. It seems to be a bit doubly confusing at the moment as the class is used on the previous version of this and has a style.

One of the things it'd have been useful for is making assertions about whether this is hidden or not. Outside of a rendered browser testing environment (JS Chrome) this is a bit tricky so we have to assert on selectors (e.g. `expect(response.body).not_to have_selector("div.covid-timeline__item:not(.covid-timeline__item--hidden).not(#nation-england)")` - which seems to look a bit odd if we do `expect(response.body).not_to have_selector("div.js-covid-timeline:not(.covid-timeline__item--hidden).not(#nation-england)")`)

Reply:
I think this has been resolved now.